### ISSUE TITLE
IS-05-01 test staging SDP file

### ISSUE THREAD

**Issue by garethsb-sony** [AMWA-TV/nmos-testing#341]
As far as I can see `IS0501Test` tests PATCH /staged on RTP Receiver with `"transport_params"`, but doesn't test `"transport_file"`. 

It would be good to enhance the testing to include testing when _only_ `"transport_file"` is included, without `"transport_params"`, and also correct behaviour when both are included in the same PATCH together.

**Comment by andrewbonney** [comment]
Agreed. Hopefully for a limited test case we can ensure that the 'transport_file' parameters correctly transition into 'transport_params'.
